README.md

```diff
@@ -1,31 +1,312 @@
+
+- `curl`
+- `shuf`
+- `sleep`
+
+#### Command Syntax
+```
+./simple-beacon.sh <IP or FQDN>
+```
+`IP or FQDN`: The IP address or FQDN of the server.
+
+#### Example Commands
+```
+./simple-beacon.sh activecountermeasures.com
+```
+Simulates a jittered beacon by sending HTTP requests to activecountermeasures.com
+
+<p align="right"><a href="#table-of-contents">Back to Top</a></p>
+
+</br>
+</br>
+
+## Python Script Pairs
+The scripts within the `python-scripts` directory work in pairs. `tcp-client.py` works with `tcp-server.py`, and `udp-client.py` works with `udp-server.py`. Unlike the other threat tools, these scripts can send a beacon to more than one server. Both client scripts will require editing the code so the IP addresses match your server IP(s). You can also edit the code to configure the server port for the C2 channel, the payload size, and the beaconing interval if you disagree with the defaults.
+
+Default Server Port: 9000</br>
+Default Payload Size: 0 - 1200 bytes</br>
+Default Beaconing Interval: 30 - 60 seconds
+
+### Dependencies
+This script will require the following tools be installed on the system running the script:
+
+- `python3`: Interpreter to run the script. Most systems have this pre-installed.
+- `pycryptodome`: Python cryptographic library.
+- `pip3`: Needed to install `pycryptodome` if it is not already installed.
+
+#### Installing `pycryptodome`
+If your system does not have `pycryptodome` installed, follow these instructions:
+
+1. Install the Python package manager, `pip3`.
+1. Run the following command:
+```
+pip3 install pycryptodome
+```
+
+### Configurations
+
+#### Configuring the Client Files
+1. On the client device, open `tcp-client.py` or `udp-client.py` in a text editor such as nano.
+1. Navigate to the line starting with `server =` followed by a list of IP addresses and replace them with the IP Address(es) of your server(s).
+1. (optional) Change the server port at the line with `SERVER_PORT = 9000`. (If you change the port number, you *must* edit the corresponding `-server.py` file to match.)
+1. (optional) Change the payload size range at the line with `m = randint(0, 1200)`.
+1. (optional) Change the beaconing interval at the line with `rsleep = random.randint(30, 60)`.
+1. Save the changes and exit.
+
+#### Configuring Server Files (optional)
+NOTE: This is only necessary if you changed the server port number in a `-client.py` script. For example, if you changed the server port in `tcp-client.py`, you *must* edit `tcp-server.py` script to match.
+1. On the server device, open the corresponding `-server.py` file in a text editor such as nano.
+1. Change the server port number at the line with `SERVER_PORT = 9000` to match the number you chose in the corresponding `-client.py` file.
+1. Save the changes and exit.
+
+### Running the Script Pairs
```

Comment:
In this section, we need to ensure that the user runs the server before the client in each case. Strictly following along with the instructions as they are now results in the client printing the destination IP address and exiting since the server isn't already running.